jekyll/_layouts/classic-docs.html

```diff
@@ -48,12 +53,12 @@
         <aside class="full-height">
           <div class='full-height-sticky'>
             {% if page.suggested != false %}
-            <div id="helpful-resources">
-              <h5 class="helpful-heading">Helpful resources</h5>
-              <div class="helpful-indent">
+            <div id="related-resources">
+              <h5 class="related-heading">Helpful resources</h5>
+              <div class="related-indent">
                 <ul class="section-nav">
                   {% for item in page.suggested %}
-                  <li class="helpful-entry helpful-h2">
+                  <li class="related-entry related-h2">
```

Comment:
You should not have these changes in this PR. Make sure to rebase master, please 🙏